src/main/resources/GoogleApps/GoogleAppsConfigSheet.vm

```diff
@@ -13,82 +13,95 @@ $xwiki.ssx.use('GoogleApps.GoogleAppsConfigClass')
     $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}")##
 #end
 #macro(tp $n $p)##
-    $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}",$p)##
+    #set($prf="GoogleApps.GoogleAppsConfigClass_")
+    #if($n.startsWith("googleapps"))#set($prf="")#end
+    #set($msg=$services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}",$p))##
+    $stringtool.replacePattern($msg, '\[\[([^>]*)>>([^\]]*)\]\]', '<a href="$2">$1</a>')##
```

Comment:
Here it would be really useful a comment explaining the meaning of this regexp you are using.

Reply:
Good point, I've done so.
Note that we loose the external-link annotation doing this. I find this to be good.